scripts/check-aur-package.sh

```
@@ -0,0 +1,191 @@
+}
+
+require_equal "$pkgname" "modfetch-bin" "pkgname"
+require_equal "$_pkgname" "modfetch" "_pkgname"
+require_equal "$pkgver" "$expected_pkgver" "pkgver"
+require_equal "${pkgrel}" "1" "pkgrel"
+
+require_file_contains "$pkgbuild" "arch=('x86_64' 'aarch64')" "arch must include x86_64 and aarch64"
+require_file_contains "$pkgbuild" "license=('MIT')" "license must be MIT"
+require_file_contains "$pkgbuild" "provides=('modfetch')" "provides must include modfetch"
+require_file_contains "$pkgbuild" "conflicts=('modfetch')" "conflicts must include modfetch"
+require_file_contains "$pkgbuild" 'source=("LICENSE::https://raw.githubusercontent.com/jxwalker/modfetch/v${pkgver}/LICENSE")' "license source mismatch"
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)